dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/s
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```

Comment:
you're stabilising an ebuild, this isn't allowed by users in general. Please speak to the maintainer such that he opens a STABLEREQ bug for it.